kikar_hamedina/templates/layouts/generic-template.html

```diff
@@ -21,7 +21,7 @@
     <meta name="License" content="CC-by-sa-3.0 Attribution-NonCommercial-ShareAlike 3.0">
     {% block social_share %}
         <!-- Open graph meta tags supported by facebook, whatsapp, linkedin and more -->
-        <meta property="og:image" content="http://kikar.org/static/media/kikar_hamedina_logo.png"/>
+        <meta property="og:image" content="http://kikar.org/static/media/kikar_hamedina_logo_900_472.png"/>
```

Comment:
@alonisser  this should be used using {{ STATIC_URL }} or {% static %} tags

e.g.             <img src="{{ STATIC_URL }}media/kikar_hamedina_logo.png" style="max-width: 100%; height: auto">